src/view/sidebar.inc

```diff
@@ -4,13 +4,13 @@
     <div class="sidebar-buttons">
       <h2 class="visually-hidden">{{'Sidebar listing: list and traverse vocabulary contents by a criterion' | trans}}</h2>
 
+      <h3 class="visually-hidden" aria-label="Sidebar tabs">{{ view_trans_text[1] | trans }}</h3>
```

Comment:
Is the `aria-label` really necessary? The `h2` element two lines above already provides a label/explanation for screen readers.

In any case it is misplaced, because this `h3` element introduces a specific tab (for example the hierarchy) and not tabs in general. So if the aria-label is needed, it should be moved to the h2 element above.

Also, the text "Sidebar tabs" should be translated using the `trans` filter in Twig.

Reply:
- Removed unnecessary H3 line
- Simplified translation infrastructure by replacing array usage with plain string values
- The copy function of the concept page occasionally produces errors on the GUI test side, but all tests pass on the command line